### PR TITLE
fix docs about syspath_prepend

### PR DESCRIPTION
**Pull request by graingert**
eg it does not modify $PATH

- [x] Target the `master` branch for bug fixes, documentation updates and trivial changes.
- [ ] Target the `features` branch for new features and removals/deprecations.
- [ ] Include documentation when adding new features.
- [ ] Include new tests or update existing tests when applicable.

Unless your change is trivial or a small documentation fix (e.g.,  a typo or reword of a small section) please:

- [ ] Create a new changelog file in the `changelog` folder, with a name like `<ISSUE NUMBER>.<TYPE>.rst`. See [changelog/README.rst](https://github.com/pytest-dev/pytest/blob/master/changelog/README.rst) for details.
- [ ] Add yourself to `AUTHORS` in alphabetical order;

### CODE REVIEW

File: doc/en/monkeypatch.rst [hunk 1]
```diff
@@ -46,10 +46,13 @@ environment variable is missing, or to set multiple values to a known variable.
 :py:meth:`monkeypatch.setenv` and :py:meth:`monkeypatch.delenv` can be used for
 these patches.
 
-4. Use :py:meth:`monkeypatch.syspath_prepend` to modify the system ``$PATH`` safely, and
+4. Use `monkeypatch.setenv("PATH", value, prepend=True)` to modify ``$PATH``, and
```
Comment (graingert) [inline doc/en/monkeypatch.rst]:
```suggestion
4. Use ``monkeypatch.setenv("PATH", value, prepend=True)`` to modify ``$PATH``, and
```

**Comment by nicoddemus** [comment]
Great, thanks!

File: doc/en/monkeypatch.rst [hunk 2]
```diff
@@ -46,10 +46,13 @@ environment variable is missing, or to set multiple values to a known variable.
 :py:meth:`monkeypatch.setenv` and :py:meth:`monkeypatch.delenv` can be used for
 these patches.
 
-4. Use :py:meth:`monkeypatch.syspath_prepend` to modify the system ``$PATH`` safely, and
+4. Use ``monkeypatch.setenv("PATH", value, prepend=True)`` to modify ``$PATH``, and
```
Comment (graingert) [inline doc/en/monkeypatch.rst]: @nicoddemus woops this should be `prepend=":"`

Comment (blueyed) [reply doc/en/monkeypatch.rst]: @graingert 
Can you do a followup PR then, please?

Comment (graingert) [reply doc/en/monkeypatch.rst]: Already done

**Pull request merged**